packages/messaging/README.md

```diff
@@ -47,16 +47,15 @@ iOS
 
 ```objectivec
 @import AEPCore;
-@import AEPLifecycle;
```

Comment:
If we are taking out the lifecycle, we need to remove the lifecycleStart in the sample below.  Same for Android and iOS.